pylintrc

```diff
@@ -393,7 +393,7 @@ ignored-classes=SQLObject, optparse.Values, thread._local, _thread._local
 # List of members which are set dynamically and missed by pylint inference
 # system, and so shouldn't trigger E1101 when accessed. Python regular
 # expressions are accepted.
-generated-members=REQUEST,acl_users,aq_parent
+generated-members=REQUEST,acl_users,aq_parent,argparse.Namespace
```

Comment:
When I revert this change I don't see a problem (locally). Problem on CI?

Reply:
I think this might become problematic at some point. Perhaps it was in an earlier version but doing `namespace.setting` will raise a `no-member` for `setting` on `namespace`. I could remove this for now, but we'll likely need it at some point.